pkgs/development/python-modules/picos/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitLab
+, numpy
+, cvxopt
+}:
+
+buildPythonPackage rec {
+  pname = "picos";
+  version = "1.2.0";
+
+  src = fetchFromGitLab {
+    owner = "picos-api";
+    repo = "picos";
+    rev = "b9c3105b31166c307296dee6e0c26cf6a0ab8723";
+    sha256 = "018xhc7cb2crkk27lhl63c7h77w5wa37fg41i7nqr4xclr43cs9z";
+  };
+
+  propagatedBuildInputs = [
+    numpy
+    cvxopt
+  ];
+
+  doCheck = false;
+
+  meta = with lib; {
+    description = "User friendly Python API to several conic and integer programming solvers, very much like YALMIP or CVX under MATLAB";
+    homepage = https://gitlab.com/picos-api/picos;
+    license = licenses.gplv3;
```

Comment:
```suggestion
    license = licenses.gpl3;
```

Reply:
Thanks, I don't know why I didn't get any errorr for that. Should I merge the two commits into a single one, or is it better to keep these two commits ?

Reply:
Yeah, it'd be nicer to have just a single commit.
If you want to keep him, you can add him like [this](https://help.github.com/en/articles/creating-a-commit-with-multiple-authors).